src/project-css/app/Console/Commands/setCmsCollection.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use App\Helpers\CollectionHelper;
+
+class setCmsCollection extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'collection:cms:set {collectioname}';
+
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Set Collection for CMS';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $helper = new CollectionHelper;
+
+        // verify collection exists
+        if ($helper->isCollection($this->argument('collectioname')) == false) {
+            return $this->error('Collection ' . $this->argument('collectioname') . ' Doesn\'t Exist');
+        }
+
+        // Using Collection Helper set collection cms option
+        $helper->setCMS($this->argument('collectioname'), true);   
+
+        $this->info('Collection is set as CMS');        
```

Comment:
No return defined, what is doing the return or is this a procedural method that just enacts other methods? If so it should be @abstract